.github/workflows/test.yml

```
@@ -54,7 +53,12 @@ jobs:
         with:
           environments: dev
           pixi-version: v0.42.1
-          cache: true
 
       - name: Run tests
         run: pixi run --environment dev test --show-capture=all -s -vv
+      
+      - name: Upload lockfile as artifact
+        uses: actions/upload-artifact@v2
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)